src/DynamoCore/Configuration/PreferenceSettings.cs

```diff
@@ -48,12 +48,14 @@ internal static void CopyProperties(this PreferenceSettings source, PreferenceSe
     /// </summary>
     public class PreferenceSettings : NotificationObject, IPreferences, IRenderPrecisionPreference, IDisablePackageLoadingPreferences, ILogSource, IHideAutocompleteMethodOptions
     {
+        internal static PreferenceSettings dynamoModelRuntimePreferenceSettings;
```

Comment:
Why do we still need this?

Reply:
no we don't... I started on the wrong base branch